satellite/gracefulexit/endpoint_test.go

```diff
@@ -396,6 +396,66 @@ func TestFailureUplinkSignature(t *testing.T) {
 	})
 }
 
+func TestSignExitCompleted(t *testing.T) {
```

Comment:
nit - maybe theses tests are better to be placed in the `signing` package?

Reply:
I agree. Moved